packages/app-store/_utils/getCalendar.ts

```diff
@@ -12,7 +12,14 @@ export const getCalendar = async (credential: CredentialPayload | null): Promise
   if (calendarType?.endsWith("_other_calendar")) {
     calendarType = calendarType.split("_other_calendar")[0];
   }
-  const calendarApp = await appStore[calendarType.split("_").join("") as keyof typeof appStore]();
+  const calendarAppImportFn = appStore[calendarType.split("_").join("") as keyof typeof appStore];
+
+  if (!calendarAppImportFn) {
```

Comment:
This ensures that a missing import for an app doesn't crash the booking flow.
No app should be important enough to crash the core booking flow.